framework/update/internal/platform/win/winupdateinstaller.cpp

```
@@ -0,0 +1,281 @@
+    return SUCCEEDED(task->get_Enabled(&enabled)) && enabled != VARIANT_FALSE;
+}
+
+QString registeredValue(const std::wstring& appId, const wchar_t* name)
+{
+    HKEY key = nullptr;
+    if (::RegOpenKeyExW(HKEY_LOCAL_MACHINE, muse::update::win::registryKeyPath(appId).c_str(), 0,
+                        KEY_QUERY_VALUE | KEY_WOW64_64KEY, &key) != ERROR_SUCCESS) {
+        return QString();
+    }
+
+    wchar_t buffer[1024] = { 0 };
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)